insalan/payment/tokens.py

```diff
@@ -61,7 +61,6 @@ def obtain_token(self, secret=None):
                     "client_secret": c_secret,
                     "grant_type": grant_type,
                 },
-                timeout=1,
```

Comment:
We should always have a timeout, lest we risk having the backend thread indefinitely hang. You could actually bump this higher instead of removing it (spoiler: it's a linter warning to use requests get/post/whatever methods without a timeout)

Reply:
I think browsers usually use something around a minute?

Reply:
That's a back request so the front will time out but it don't think the back will.